CONTRIBUTING.md

```diff
@@ -57,6 +57,22 @@ Note: `release/x.y` or `hotfix/x.y.z` will be merged back in `master` after a ne
 
 [git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
 
+# Subtree'd library projects
+
+A number of library dependencies are managed as separate open source projects and are git-subtree'd into the WordPress Android app source tree. Use the following command to updated (pull latest) from their respective repos:
+
+        $ git subtree pull --prefix libs/library_name https://github.com/wordpress-mobile/WordPress-Library_Name-Android.git develop
```

Comment:
Hey Maxime. I intentionally left out both the `--squash` and the info to push. Lemme elaborate.

About the `--squash`:
* We typically merge to the main repo (instead of rebasing for example) and the pull from the external project will look more or less like such a merge anyway
* Merging an external project it's probably more important to keep an eye on (read: see it on git log) what got merged. The squashed pull is very obscure at that.
* This being a manual procedure, one can add that option at the time of execution, perhaps deciding in depending on the commit count of the change
* It's a matter of preference anyway. More than happy to add the `--squash` if the consensus is towards that :)

About the push info:
In my mind, there's going to be 2 groups of people: One with the people that has commit access and one with the contributors without commit access. The `CONTRIBUTING.md` file is mainly oriented towards the second group from which we expect contributions via Pull Requests and that's why I didn't include push info.
For the first group, I think we can find a different place/file to document the push command. As always, happy to add it here if we decide so :)